pubspec.yaml

```diff
@@ -1,24 +1,18 @@
 name: grinder
-# This version must be updated in tandem with the lib/src/cli.dart version.
```

Comment:
I'd like to keep this comment - it helps me remember which projects are ones where I need to update source files in tandem with the pubspec files.